core/site_scons/site_tools/micropython/__init__.py

```diff
@@ -26,9 +26,11 @@ def generate_frozen_module(source, target, env, for_signature):
         # replace "utils.BITCOIN_ONLY" with literal constant (True/False)
         # so the compiler can optimize out the things we don't want
         btc_only = env['bitcoin_only'] == '1'
+        zcash_shielded = env['zcash_shielded'] == '1'
         interim = f"{target[:-4]}.i"  # replace .mpy with .i
         sed_scripts = " ".join([
             rf"-e 's/utils\.BITCOIN_ONLY/{btc_only}/g'",
+            rf"-e 's/utils\.ZCASH_SHIELDED/{zcash_shielded}/g'",
```

Comment:
I have no idea what is going on these lines. What should I do?

append `rf"-e 's/^\s+ZCASH_SHIELDED/# \0/'",` ?

Reply:
that should do it, yes.

these are individual commands to `sed`, the leading `r` marks it as a regexp string so that `\` are literal
so the selected line will replace all instances of `utils.ZCASH_SHIELDED` with a literal `True`, and the proposed new line will replace `   ZCASH_SHIELDED,` with `# ZCASH_SHIELDED,` (we must not change number of lines, otherwise our coverage tool misrepresents which lines are covered)

Reply:
thank you for explanation

https://github.com/trezor/trezor-firmware/compare/23beeecc618cb39e0fc1baee36b5efff19700b33..e73a02a81d7dd6b666205117e5a5dccc2e41c3f7